supysonic/api/__init__.py

```diff
@@ -24,6 +24,18 @@
 api = Blueprint("api", __name__)
 
 
+def api_routing(endpoint):
+    def decorator(func):
+        viewendpoint="{}.view".format(endpoint)
+        @api.route(endpoint, methods=["GET", "POST"])
+        @api.route(viewendpoint, methods=["GET", "POST"])
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            return func(*args,**kwargs)
+        return wrapper
```

Comment:
Rather than using the `route()` decorator you could call [`add_url_rule()`](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Blueprint.add_url_rule). That would spare the wrapper and you could directly return the `func` callable.

`route()` actually calls `add_url_rule()`. You can fin its code [here](https://github.com/pallets/flask/blob/1.1.x/src/flask/blueprints.py#L272)